py2-v1/setup.sh

```diff
@@ -58,7 +58,13 @@ if [ -z ${OpenCL} ]; then
     LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${SROOTBASE}/../distrib/OpenCL_$OS_ARCH/lib/$OS_ARCH
 fi
 
-for name in SROOTBASE SROOT SITE_CMAKE_DIR PATH MANPATH PKG_CONFIG_PATH LD_LIBRARY_PATH PYTHONPATH OS_ARCH GCC_VERSION GOTO_NUM_THREADS OPENCL_VENDOR_PATH CC CXX FC
+# PGPLOT
+if [ -f $SROOT/share/grfont.dat ]; then
+    PGPLOT_DIR=${SROOT}/share/pgplot
+    PGPLOT_DEV=/xwin
+fi
```

Comment:
Why do these need to be set in the global environment?

Reply:
The documentation says these need to be set for any code to work.  The variable is used within the pgplot source code to find the grfont.dat and rgb.txt files.  Also, whoops, just noticed the file path it's checking is wrong...